test/stress_tests/test_dead_actors.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import sys
+
+import ray
+
+ray.init()
```

Comment:
This needs to be `ray.init(redis_address="localhost:6379")` in order to attach to the cluster.